de.metas.adempiere.adempiere/base/src/main/java/de/metas/event/impl/EventBus.java

```diff
@@ -199,52 +194,58 @@ public void postEvent(@NonNull final Event event)
 	}
 
 	@Override
-	public boolean isDestroyed()
+	public void postObject(@NonNull final Object obj)
 	{
-		return destroyed;
+		final String json = SimpleObjectSerializer.get().serialize(obj);
```

Comment:
we also have `de.metas.util.JSONObjectMapper`. 
Which seems to be the same, it's just aiming at being usable beyond events.
(like, in DataEntry where we store the actual entered data as JSON)

@teosarca @TheBestPessimist wdyt, maybe delete SimpleObjectSerializer and use JSONObjectMapper (in this case `JSONObjectMapper.forClass(Oject.class)` instead? 
Or maybe let SimpleObjectSerializer  deletegate to JSONObjectMapper.
I think it would be good if e.g. date values would be serialized the same way everywhere within mf.

Reply:
+1

Reply:
SimpleObjectSerializer shall call JSONObjectMapper... 
... or better: get rid of SimpleObjectSerializer!
wdyt?

Reply:
If we're to use `JSONObjectMapper` the code looks ugly imo:
```java
final JSONObjectMapper<Object> jsonObjectMapper = JSONObjectMapper.forClass((Class<Object>)obj.getClass());
final String json = jsonObjectMapper.writeValueAsString(obj);
```
since `JSONObjectMapper` expects a type, whereas `SimpleObjectSerializer` doesn't.

For now i think we should keep the code as it is until we improve the code.

To signify the possible change though i marked `SimpleObjectSerializer` as `@Deprecated`.